packages/console/src/consts/logs.ts

```diff
@@ -0,0 +1,21 @@
+type LogEventTitle = Record<string, string>;
+
+export const logEventTitle: LogEventTitle = Object.freeze({
+  RegisterUsernamePassword: 'Register with username and password',
+  RegisterEmailSendPasscode: 'Register with email (send passcode)',
+  RegisterEmail: 'Register with email',
+  RegisterSmsSendPasscode: 'Register with SMS (send passcode)',
+  RegisterSms: 'Register with SMS',
+  RegisterSocialBind: 'Bind social account',
+  RegisterSocial: 'Register with social account',
+  SignInUsernamePassword: 'Sign in with username and password',
+  SignInEmailSendPasscode: 'Sgin in with email (send passcode)',
+  SignInEmail: 'Register with email',
+  SignInSmsSendPasscode: 'Sgin in with SMS (send passcode)',
+  SignInSms: 'Sign in with SMS',
+  SignInSocialBind: 'Sign in with social related account',
+  SignInSocial: 'Sign in with social account',
+  CodeExchangeToken: 'Exchange token by auth code',
```

Comment:
nit: better replace `auth` with `authorization`?

Reply:
It will be too long in table.